update-changelog/writeDocsFiles.ts

```diff
@@ -14,8 +14,7 @@ export async function writeDocsFiles({ version, builder }: DocsReleaseNotesWrite
 
 	const content = `+++
 title = "${title}"
-[_build]
-list = false
+hide_menu = false
```

Comment:
Should this not be true?

Reply:
If my understanding is correct the Grafana docs no longer use  `[_build]
list = true` in the hugo templates at all. We just need hide_menu set to false now.

For reference: https://github.com/grafana/website/pull/4809 and https://github.com/grafana/website/pull/7390

Reply:
`hide_menu` should be set to true, so the page doesn't show in the left menu.

We need `_build.list` set to true (or removed) in order to query the release notes for the docs homepage.

Reply:
Ahhh I understand now. Thank you for clarifying.